client/signup/jetpack-connect/authorize-form.jsx

```diff
@@ -167,7 +168,7 @@ const LoggedOutForm = React.createClass( {
 		return (
 			<LoggedOutFormLinks>
 				<LoggedOutFormLinkItem href={ login( { redirectTo } ) }>
-					{ this.translate( 'Already have an account? Sign in' ) }
+					{ i18n.translate( 'Already have an account? Sign in' ) }
```

Comment:
We should be calling on the `translate` prop passed by the `localize` higher-order component, instead of on the `i18n` default export. Reflecting on this, I think there's probably no difference in that merely by wrapping the component in `localize`, we can be sure that it accounts for [changes emitted by the underlying localization store](https://github.com/Automattic/i18n-calypso/blob/26d4474be91364d83adaca763ae5ba88ffce15a9/lib/localize/index.js#L24). But as far as conventions for higher-order components (and, more importantly, testability), it's better to reference this by the incoming prop.